localstack/services/edge.py

```diff
@@ -516,6 +513,7 @@ def get_service_port_for_account(service, headers):
 
 
 PROXY_LISTENER_EDGE = ProxyListenerEdge()
+router: Router[Handler] = Router(dispatcher=handler_dispatcher())
```

Comment:
Maybe we could add a line of docs on what this is for (that this integrates the new routing framework)?